### PR TITLE
Add alpine versions of images.

### PR DESCRIPTION
**Pull request by slashrsm**
Image size 27MB vs 150MB.

### CODE REVIEW

**Comment by zaporylie** [comment]
Failing due to missing alpine for 5.4 and missing `apt-get`. You may want to use `apk` instead

**Comment by slashrsm** [comment]
Green.

File: 5.6-alpine/fpm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+VOLUME /var/www/html
+
+RUN apk add --no-cache --virtual .dd-build-deps libpng-dev libjpeg-turbo-dev postgresql-dev $PHPIZE_DEPS \
+   && docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
+   && docker-php-ext-install gd mbstring pdo pdo_mysql pdo_pgsql zip \
+   && docker-php-ext-install opcache \
+   && pecl install redis-2.2.8 \
+   && docker-php-ext-enable redis \
+   && apk add --no-cache sudo git libpng libjpeg libpq \
```
Comment (zaporylie) [inline 5.6-alpine/fpm/Dockerfile]: I don't think we need git or sudo for *-fpm images (5.6, 7.0, 7.1).

**Review by zaporylie — CHANGES_REQUESTED** [review]
I think it's almost good to go. Just small issue I've mentioned above - *-fpm images should be free sudo or git.

**Comment by zaporylie** [comment]
Oh... comment was actually added below 😄  still learning how to use new Github review tools.

File: 5.6-alpine/fpm/Dockerfile (hunk 1 above)
Comment (slashrsm) [reply 5.6-alpine/fpm/Dockerfile]: Makes sense. Done.

**Comment by slashrsm** [comment]
Test fails seems to be unrelated since they happened in PHP 5.4 image, which we didn't touch here.

**Comment by zaporylie** [comment]
I have restarted that job. Feel like it's Travis infra issue.

Anyway - this stats looks impressive:
7.1-alpine          latest              bfaabef70542        Less than a second ago   96.38 MB
7.1-alpine/fpm      latest              ea039aa6c4dc        Less than a second ago   89.67 MB
7.1                 latest              75cc51bad7d0        Less than a second ago   434.7 MB
7.1/fpm             latest              3f0f00c4af23        Less than a second ago   408.9 MB

**Comment by slashrsm** [comment]
Agreed! 
What can we do regarding Travis? Wait?

**Comment by zaporylie** [comment]
Just got alert from Docker Hub that 5.4-cli is failing there too. I will investigate later this weekend what could cause that error.

**Review by zaporylie — APPROVED** [review]

**Comment by zaporylie** [comment]
Ok, so "wait" was enough :) Just rerun failing build and now it's ok :) I'm gonna prepare tags on hub.docker.com and merge that pull request as soon as it's done.

**Comment by slashrsm** [comment]
Yay! Thanks @zaporylie.

**Pull request merged**